src/command_modules/azure-cli-feedback/setup.py

```diff
@@ -25,6 +25,7 @@
 
 DEPENDENCIES = [
     'applicationinsights',
```

Comment:
> 'applicationinsights', [](start = 3, length = 23)

Is this an old hand authored SDK? The swagger for insights is currently in PR status being reviewed.